### PR TITLE
[DO NOT MERGE] pulseeffects: update to 4.3.4

### PR DESCRIPTION
**Pull request by steinex**
One issue left:

```
$ PULSEEFFECTS_DEBUG=1 pulseeffects
(gst-plugin-scanner:12671): GStreamer-WARNING **: 01:14:33.506: Failed to load plugin '/usr/lib/gstreamer-1.0/libgstdtls.so': /usr/lib/gstreamer-1.0/libgstdtls.so: undefined symbol: DTLS_method
```

I suspect this is because `gst-plugins-bad1` is build with LibreSSL, thus missing that symbol? Any input how to go from here would be appreciated!

### CODE REVIEW

**Comment by cr6git** [comment]
https://travis-ci.org/void-linux/void-packages/jobs/426313294#L401
````
$ common/travis/xlint.sh
srcpkgs/libebur128/template:8: license 'BSD', but no use of vlicense
srcpkgs/libebur128/template:8: license 'MIT', but no use of vlicense
````

File: srcpkgs/pulseeffects/template [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
- gst-plugins-good1 pulseaudio python3-gobject python3-numpy python3-scipy"
+hostmakedepends="itstool pkg-config"
+makedepends="boost-devel glib-devel gsettings-desktop-schemas-devel
+ gst-plugins-bad1-devel gtkmm-devel libbs2b-devel libebur128-devel
+ lilv-devel pulseaudio-devel python-gobject-devel sratom-devel"
+depends="ConsoleKit2 calf gst-plugins-bad1 gst-plugins-good1 lilv pulseaudio
+ python3-gobject python3-scipy"
 short_desc="Sound effects for Pulseaudio applications"
 maintainer="cr6git <quark6@protonmail.com>"
 license="GPL-3"
```
Comment (cr6git) [inline srcpkgs/pulseeffects/template]: `license="GPL-3.0-or-later"`

**Comment by Johnnynator** [comment]
Can you try it while using this patch for `gst-plugins-bad1`
```
--- ext/dtls/gstdtlsagent.c	2018-03-23 21:45:15.000000000 +0100
+++ -	2018-09-09 14:05:57.663350425 +0200
@@ -176,7 +176,7 @@
 
   ERR_clear_error ();
 
-#if OPENSSL_VERSION_NUMBER >= 0x1000200fL
+#if OPENSSL_VERSION_NUMBER >= 0x1000200fL && !defined(LIBRESSL_VERSION)
   priv->ssl_context = SSL_CTX_new (DTLS_method ());
 #else
   priv->ssl_context = SSL_CTX_new (DTLSv1_method ());
```

**Comment by steinex** [comment]
If I change the patch to `&& !defined(LIBRESSL_VERSION_NUMBER)` this fixes the problem. I will open another PR to include this patch in `gst-plugins-bad1`, thanks!

**Comment by steinex** [comment]
@cr6git why did you do this? The patch for `gst-plugins-bad1` is not even in as I didn't have the time yet plus there are still other issues that popped up regarding pulseeffects (segfaults, not all effect plugins available). This is in an unusable state and should be reverted.

**Comment by cr6git** [comment]
> @cr6git why did you do this?

because I'm stupid... :unamused: 
this package already caused enough trouble, please could you adopt / maintain it?

**Comment by steinex** [comment]
> because I'm stupid... 

Hey, no worries. I was just a bit upset to see this happen even though the pull request being labeled with "DO NOT MERGE". Sorry for being a bit harsh earlier, I didn't mean to rub you the wrong way.

> this package already caused enough trouble, please could you adopt / maintain it, @steinex ?

I know exactly what you mean and I hate myself already for going down this rabbit hole ;-). Even with the patch above applied there are still issues left that need to be sorted out before the version bump can happen. But I will continue to work on it as time allows and I will consider updating `maintainer=` to myself if I feel confident enough that I can manage this.

I will open another PR in the next days, trying to get this to 4.3.4 finally.

**Comment by cr6git** [comment]
@steinex update to 4.3.5 :racehorse:

**Pull request closed without merging**